ship_it/__init__.py

```diff
@@ -5,11 +5,17 @@
 from ship_it.manifest import Manifest, get_manifest_from_path
 from ship_it import cli
 from ship_it.virtualenv import VirtualEnvPackager
+import subprocess
+import sys
 
 
 def validate_path(path_to_check):
     assert path.isabs(path_to_check) and path.isfile(path_to_check)
 
+def version_from_setup_py(setup_py_path):
+    out = subprocess.check_output([sys.executable, setup_py_path, '--version'])
```

Comment:
I could see an argument for using ``invoke`` to do this same thing, since I'm slightly more confident that they'll make sure everything works fine across python versions, I'll test locally.